src/normalize/normalize.py

```diff
@@ -28,13 +27,9 @@
 
 
 def check_config(config: dict):
-    assert config["raw_prefix"]
-    assert config["normalize_prefix"]
-
-    assert config["feed_id"]
-
-    assert config["normalize_config"]["start_date"]
-    assert config["normalize_config"]["state_file"]
+    assert config["s3_bucket"]["uri"]
+    assert config["s3_bucket"]["public_key"]
+    assert config["s3_bucket"]["secret_key"]
```

Comment:
I think we should be validating the normalize configs here right?

Reply:
Resolved as we went thru in the call that global_config will not have normalize config